Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.h

```diff
@@ -69,6 +69,8 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qMRMLSubjectHierarchyTreeV
   Q_PROPERTY(QString nameFilter READ nameFilter WRITE setNameFilter)
   Q_PROPERTY(QStringList nodeTypes READ nodeTypes WRITE setNodeTypes)
   Q_PROPERTY(QStringList hideChildNodeTypes READ hideChildNodeTypes WRITE setHideChildNodeTypes)
+  Q_PROPERTY(vtkIdType currentItem READ currentItem WRITE setCurrentItem DESIGNABLE false)
+  Q_PROPERTY(QList<vtkIdType> currentItems READ currentItems WRITE setCurrentItems DESIGNABLE false)
```

Comment:
You could update the PR to mark `setCurrentItem` and `setCurrentItems` as Q_INVOKABLE ?

Reply:
Those are slots, so are already callable.